t/plugin/proxy-rewrite3.t

```diff
@@ -455,3 +454,51 @@ passed
 GET /echo HTTP/1.1
 --- response_headers
 test: test_in_set
+
+
+
+=== TEST 21: set route (test if X-Forwarded-Port can be set before proxy)
```

Comment:
It seems the newly added test is not relative to the ai module? Can it pass without your modification?

Reply:
`var_x_forwarded_port` is set in `set_upstream_headers`, `set_upstream_headers` is called in `handle_upstream`, and `handle_upstream` is replaced in the ai module.
It will fail in the ci env if the ai module is triggered.